packages/terra-form/src/NumberField.jsx

```diff
@@ -107,6 +113,7 @@ const NumberField = (
     label={label}
     labelAttrs={labelAttrs}
     help={help}
+    htmlFor
```

Comment:
You are right. I missed it in this pull request, thus the failing build. Added it here.

https://github.com/cerner/terra-core/pull/511/commits/41218bd352a23895f9d8866b73d23100126d0faa